README.md

```diff
@@ -109,17 +109,41 @@ and you can repeat the search performed above using its interface.
 
 The format of the log is described in detail in ["Navigating the Merkle Logs"](https://docs.datatrails.ai/developers/developer-patterns/navigating-merklelogs/) (note: this material is not released yet)
 
+# Verifying a single event
+
+An example of verifying the following single event using api response data.
+
+https://app.datatrails.ai/archivist/v2/publicassets/87dd2e5a-42b4-49a5-8693-97f40a5af7f8/events/a022f458-8e55-4d63-a200-4172a42fc2aa
+
+We use a publicly attested event so that you can check the event details directly.
+
+EVENT_ID=publicassets/87dd2e5a-42b4-49a5-8693-97f40a5af7f8/events/a022f458-8e55-4d63-a200-4172a42fc2aa
+DATATRAILS_URL=https://app.datatrails.ai
+PUBLIC_TENANT_ID=tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
+
+curl -sL $DATATRAILS_URL/archivist/v2/$EVENT_ID | veracity --url $DATATRAILS_URL/verifiabledata --tenant=$PUBLIC_TENANT_ID events-verify
+
+output will be:
+
+    verifying for tenant: tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
+    verifying: 663 334 018fa97ef269039b00 2024-05-24T08:27:00.2+01:00 publicassets/87dd2e5a-42b4-49a5-8693-97f40a5af7f8/events/a022f458-8e55-4d63-a200-4172a42fc2aa
+    leaf hash: bfc511ab1b880b24bb2358e07472e3383cdeddfbc4de9d66d652197dfb2b6633
+    OK|663 334|[aea799fb2a8c4bbb6eda1dd2c1e69f8807b9b06deeaf51b9e0287492cefd8e4c, 9f0183c7f79fd81966e104520af0f90c8447f1a73d4e38e7f2f23a0602ceb617, da21cb383d63896a9811f06ebd2094921581d8eb72f7fbef566b730958dc35f1, 51ea08fd02da3633b72ef0b09d8ba4209db1092d22367ef565f35e0afd4b0fc3, 185a9d55cf507ef85bd264f4db7228e225032c48da689aa8597e11059f45ab30, bab40107f7d7bebfe30c9cea4772f9eb3115cae1f801adab318f90fcdc204bdc, 94ca607094ead6fcd23f52851c8cdd8c6f0e2abde20dca19ba5abc8aff70d0d1, 4b6dc2ff8d608faee7be16f900d58f7ff02360db319dc68f76035890d65c8c05, 7fafc7edc434225afffc19b0582efa2a71b06a2d035358356df0a52d2256c235]
+
+The same command accepts the result of a DataTrails list events call. And the event data can be supplied as local file if desired.
+
 # General use commands
 
 * `node` - read a merklelog node
 * `nodescan` - scan a log for a particular node value
 * `diag` - print diagnostics about a massif, identified by massif index or by an mmr index
-* `ediag` - print diagnostics about an events entry in the log (currently only supports events on protected assets)
+* `events-verify` - verify the inclusion of an event, or list of events, in the tenant's merkle log
```

Comment:
I think this would be better if it was consistent with the demos we are doing?

I think the consistent terms are integrity, completeness and consistency?

Also do we need the events prefix? i understand in the future we may have non event leaf nodes on the mmr, but that sounds like a future versioning problem?

Reply:
integrity is the wrong name for what this is. We are verifying an inclusion proof. The term "inclusion proof" is used consistently everywhere else for this.

I expect us to have two basic verification actions going forward:

* verify the inclusion in the log, with *optional* check for a seal attesting that inclusion
* verify the attestation, which just checks the signature on the attestation, and optionally verifies the inclusion

The verbs that makse sense to me for that are verify-inclusion, and verify-attested. I have truncated verify-inclusion to just verify, but I can put the -inclusion back if that is clearer.
I expect these modes to apply to event data from the api.

I expect these to apply to  scitt statements

So yeah, I think event- prefix is necessary, we can (and do have aliases)

I expect scitt-verify , and scitt-attested will also become a thing

Reply:
happy if we feel we should use different terms, I think my main concern is consistent terms :)

Reply:
resolving for now :) we have decided to have a call about naming of commands and subcommands in veracity later